ansible/README.md

```diff
@@ -88,6 +88,15 @@ ansible_winrm_read_timeout_sec: 630
 
 Additional information about `winrm` variables can be found [here](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/user_guide/windows_winrm.rst#inventory-options)
 
+You may encounter a python related error
```

Comment:
We've only seen this when running the playbook from a mac, AFAIK. Until it's an issue with anything else, could you specify that this is only for mac, just in case this does unforeseen things on any other Unix system :-)